settings.py

```diff
@@ -151,7 +151,8 @@
 SESSION_ENGINE = "django.contrib.sessions.backends.cached_db"
 
 # time in minutes before forced logout due to inactivity
-INACTIVITY_TIMEOUT = 30
+INACTIVITY_TIMEOUT = 60 * 24 * 14
```

Comment:
Instead of these what if we made them local to `Domain` and had a `get_inactivity_timeout` that returned either 2 weeks or 30 min?

That won't work very well if we want different values of these for each server. (I don't think we do though?)